legend-depot-core-http/src/main/java/org/finos/legend/depot/tracing/resources/BaseResource.java

```diff
@@ -116,4 +145,9 @@ private void registerResourceApisMetrics(BaseResource baseResource)
             return resourceName;
         });
     }
+
+    protected <T> Response handle(String label, Supplier<T> supplier, Request request, Supplier<EntityTag> entityTagSupplier)
+    {
```

Comment:
why do we need this method?

Reply:
there are two handle function  that takes parameters
1. String label, Supplier<T> supplier
2. String resourceAPIMetricName, String label, Supplier<T> supplier
Created this function to handle first kind of handle